swarm/network/fetcher.go

```diff
@@ -184,7 +209,7 @@ func (f *Fetcher) run(ctx context.Context, peers *sync.Map) {
 			var err error
 			sources, err = f.doRequest(ctx, gone, peers, sources)
 			if err != nil {
-				log.Debug("unable to request", "request addr", f.addr, "err", err)
+				log.Warn("unable to request", "request addr", f.addr, "err", err)
```

Comment:
I wonder, should we do something about it at this point? Is having a `Warn` enough? I am aware that we may not be able to do much, I guess `panic` and stop the node is too radical.

Reply:
it is totally acceptable to have this when we are not connected